packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -1864,6 +1865,13 @@ abstract class RouteAware {
 /// The `settings` argument define the settings for this route. See
 /// [RouteSettings] for details.
 ///
+/// {@template flutter.widgets.RawDialogRoute}
+/// [DisplayFeatureSubScreen] is used for displaying this route on only one
```

Comment:
The documentation here and elsewhere where the `RawDialogRoute` and the `DisplayFeatureSubScreen` are used reads a little strange.

The fact that DisplayFeatureSubScreen is used is an implementation detail that's not important for the doc here. Furthermore, the concept of a sub-screen isn't really introduced to the reader of these docs.

Instead, the docs should describe what happens if the screen is divided into sub-screens by display features like hinges, etc. and how that behavior can be influenced with anchorPoint.

Reply:
Instead of somewhat awkwardly stitching together two macros here, maybe it would make sense to write this in one flowing piece of text here and use that as a single macro everywhere else. That also reduces the risk that the two macros get separate in a later refactoring.

Reply:
I made the changes you suggested. The docs no longer say that `DisplayFeatureSubScreen` is used, but I still direct the reader to look at the `DisplayFeatureSubScreen` docs if they want to further understand the specifics of how display features split the screen into sub-screens.